### ISSUE TITLE
Externalize Build Download URL

### ISSUE THREAD

**Issue by j4qfrost** [rust-skia/rust-skia#328]
I want to be able to pass the download url from the package metadata down to this line: https://github.com/rust-skia/rust-skia/blob/97114c55cbdb367a6efc2672bcf952376248e724/skia-bindings/build_support/binaries.rs#L124

The motivation is that I want to be able to create my own builds of skia and test changes quickly in a CI. I wanted to get some input before starting a pull request.

**Comment by pragmatrix** [comment]
Do you mean with "the package metadata", the metadata in a top level crate that references skia-bindings or skia-safe?

**Comment by j4qfrost** [comment]
It might be more convenient to pass it down through skia-safe to skia-bindings, which probably means both would have to be allowed.

**Comment by j4qfrost** [comment]
https://github.com/rust-skia/rust-skia/pull/332 Actually using an environment variable makes more sense.

**Comment by pragmatrix** [comment]
Support for the environment variable `SKIA_BINARIES_URL` has been released in version [0.27.3](https://github.com/rust-skia/rust-skia/releases/tag/0.27.3) today. Closing, and thank you @j4qfrost

**Comment by shore-yong** [comment]
notes for SEO purpose: use `SKIA_BINARIES_URL` for specifying address for downloading skia binaries, in case slow access 
 to GitHub release assets.

Chinese version: 从 GitHub 下载 Skia 预编译二进制文件很慢的话, 可以试试用 `SKIA_BINARIES_URL` 指定一个自己的镜像地址.

-----

Url format according to source code: https://github.com/rust-skia/rust-skia/blob/9f9f03029eab651b708a2260c116ab7a8e0fe68c/skia-bindings/build_support/binary_cache/env.rs#L22-L25

```
https://github.com/rust-skia/skia-binaries/releases/download/{tag}/skia-binaries-{key}.tar.gz
```
